apps/backend/src/user/user.service.ts

```diff
@@ -99,6 +99,9 @@ export class UserService {
       updateData.canHelpNoobs = false;
       updateData.publicDesc = '';
     }
+    if (updateData.isSchResident === false) {
+      updateData.roomNumber = null;
+    }
```

Comment:
amugy ez elvileg frontenden is meg van csinalva, de lehet megis rosszul kuldi akkor le

Reply:
hat nekem nem ugy tunt, meg az nem is tul nagy vedelem :D